examples/gnrc_border_router/Makefile.native.conf

```diff
@@ -12,8 +14,10 @@ CFLAGS += $(if $(CONFIG_KCONFIG_MODULE_DHCPV6),,-DCONFIG_DHCPV6_CLIENT_PFX_LEASE
 # -z [::1]:$PORT for each ZEP device
 TERMFLAGS += $(patsubst %,-z [::1]:%, $(shell seq $(ZEP_PORT_BASE) $(ZEP_PORT_MAX)))
 
+POSTTERM ?= killall uhcpd ; killall zep_dispatch
```

Comment:
This is a bit aggressive, why did you need to add `uhcpd` here?

Reply:
Hm I guess alternatively I could create a `start_network.sh` script for `native`

Reply:
I created a script that takes care of starting / shutting down all the deamons.
Should be much cleaner now :)